src/client/websocket/WebSocketShard.js

```diff
@@ -475,12 +475,20 @@ class WebSocketShard extends EventEmitter {
       return;
     }
     const hasGuildsIntent = new Intents(this.manager.client.options.intents).has(Intents.FLAGS.GUILDS);
-    // Step 2. Create a 15s timeout that will mark the shard as ready if there are still unavailable guilds
+    // Step 2. Create a timeout that will mark the shard as ready if there are still unavailable guilds
+    // * the timeout should be 15 seconds by default
+    // * this can be optionally be changed in the client options via fetchGuildTimeout
+    // * a timeout time of zero will skip this timeout
```

Comment:
While this is documentation in internal code, users shouldn't use a timeout of 0 without acknowledging the difference it can make